DEVELOP.rst

```diff
@@ -42,10 +42,39 @@ To build the ``cratedb-prometheus-adapter`` executable, run::
 
    go build
 
+Unit tests
+==========
+
 To run the test suite, execute::
 
-   go test -v
+    go test -v
+
+For running the linter, invoke::
+
+    gofmt -l -d .
+
+To format the files, according to what the linter says, run::
+
+    gofmt -w .
+
+Integration tests
+=================
+
+Run CrateDB and Prometheus::
 
+    docker compose --file tests/docker-compose.yaml up
+
+Run integration tests::
+
+    pytest
```

Comment:
Should mention to run `pip3 install --upgrade -r requirements-test.txt` before?

Reply:
Absolutely, thank you again. I've expanded this a bit more to also educate readers about how to set up the Python sandbox (virtualenv). See 87f8e11720c.

Reply:
May I ask you to review both spots once more and acknowledge the PR again? Thanks!

Reply:
Looks good, thanks!